### PR TITLE
fix: wrong progress indicator [AR-1671]

### PR DESCRIPTION
**Pull request by gongracr**
<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/AR-1671" title="AR-1671" target="_blank">BugAR-1671</a>  No loading indicator when uploading large file
  </td></table>
  <br />
 

----
#### PR Submission Checklist for internal contributors

- The **PR Title**
  - [x] conforms to the style of semantic commits messages¹ supported in Wire's Github Workflow²
  - [x] contains a reference JIRA issue number like `SQPIT-764`
  - [x] answers the question: _If merged, this PR will: ..._ ³

- The **PR Description**
  - [x] is free of optional paragraphs and you have filled the relevant parts to the best of your ability
----

# What's new in this PR?

### Issues
Asset messages were not being displayed correctly according to the UI designs. This PR fixes that.

### Attachments (Optional)
| Before  | After |
| ------------- | ------------- |
| Wire 2022-10-10 at 2_29 PM  | Wire 2022-10-12 at 13_26 |
----
#### PR Post Submission Checklist for internal contributors (Optional)

 - [x] Wire's Github Workflow has automatically linked the PR to a JIRA issue
----
#### PR Post Merge Checklist for internal contributors

 - [x] If any soft of configuration variable was introduced by this PR, it has been added to the relevant documents and the CI jobs have been updated.
----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.

### CODE REVIEW

**Comment by AndroidBob** [comment]
Build [1](https://10.10.124.134/job/wire-android-reloaded/job/PR-1093/1/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv0.0.1(4662059)(4662059)-dev-release.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/dev/release/com.wire.android-vv0.0.1(4662059)(4662059)-dev-release.apk)

**Comment by AndroidBob** [comment]
Build [2](https://10.10.124.134/job/wire-android-reloaded/job/PR-1093/2/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv0.0.1(4668029)(4668029)-dev-release.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/dev/release/com.wire.android-vv0.0.1(4668029)(4668029)-dev-release.apk)

File: app/src/main/kotlin/com/wire/android/ui/home/conversations/model/messagetypes/asset/AssetMessageTypes.kt [hunk 1]
```diff
@@ -64,68 +66,96 @@ internal fun MessageAsset(
...
-                            top.linkTo(parent.top)
-                            start.linkTo(parent.start)
-                            bottom.linkTo(parent.bottom)
-                        },
-                    painter = painterResource(R.drawable.ic_file),
-                    contentDescription = stringResource(R.string.content_description_image_message),
-                    colorFilter = ColorFilter.tint(MaterialTheme.wireColorScheme.badge)
-                )
+        if (assetUploadStatus == Message.UploadStatus.UPLOAD_IN_PROGRESS) {
+            // If asset is being uploaded, we don't want to show any asset metadata info
```
Comment (yamilmedina) [inline app/src/main/kotlin/com/wire/android/ui/home/conversations/model/messagetypes/asset/AssetMessageTypes.kt]: Personal taste here, but I would prefer something like removing this comment and creating a val with this condition:

`val shouldHideAssetMetadata = assetUploadStatus == Message.UploadStatus.UPLOAD_IN_PROGRESS`

Nothing blocker anyway

**Review by yamilmedina — APPROVED** [review]
LGTM compa ♠️ !

File: app/src/main/kotlin/com/wire/android/ui/home/conversations/model/messagetypes/asset/AssetMessageTypes.kt (hunk 1 above)
Comment (gongracr) [reply app/src/main/kotlin/com/wire/android/ui/home/conversations/model/messagetypes/asset/AssetMessageTypes.kt]: Not very opinionated about it, so why not?

**Comment by AndroidBob** [comment]
Build [3](https://10.10.124.134/job/wire-android-reloaded/job/PR-1093/3/) **failed**.

**Comment by AndroidBob** [comment]
Build [4](https://10.10.124.134/job/wire-android-reloaded/job/PR-1093/4/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv0.0.1(4669895)(4669895)-dev-release.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/dev/release/com.wire.android-vv0.0.1(4669895)(4669895)-dev-release.apk)

**Review by borichellow — APPROVED** [review]
LGTM

**Comment by AndroidBob** [comment]
Build [5](https://10.10.124.134/job/wire-android-reloaded/job/PR-1093/5/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv0.0.1(4672680)(4672680)-dev-release.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/dev/release/com.wire.android-vv0.0.1(4672680)(4672680)-dev-release.apk)

**Pull request merged**